src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -104,8 +105,10 @@ private static Person createEditedPerson(Person personToEdit, EditPersonDescript
         Company updatedCompany = editPersonDescriptor.getCompany().orElse(personToEdit.getCompany());
         Set<Tag> updatedTags = editPersonDescriptor.getTags().orElse(personToEdit.getTags());
         Set<Product> updatedProducts = editPersonDescriptor.getProducts().orElse(personToEdit.getProducts());
+        SupplierStatus supplierStatus = personToEdit.getStatus();
 
-        return new Person(updatedName, updatedPhone, updatedEmail, updatedCompany, updatedTags, updatedProducts);
+        return new Person(updatedName, updatedPhone, updatedEmail, updatedCompany, updatedTags,
+                updatedProducts, supplierStatus);
```

Comment:
Should the naming of supplierStatus variable be renamed to reflect that it is the updated status?